framework/scenario/tests/contract_without_macros.rs

```diff
@@ -34,7 +33,7 @@ mod module_1 {
     /////////////////////////////////////////////////////////////////////////////////////////////////
     //////// AUTO-IMPLEMENTED METHODS ///////////////////////////////////////////////////////////////
     /////////////////////////////////////////////////////////////////////////////////////////////////
-    pub trait AutoImpl: multiversx_sc::contract_base::ContractBase {}
+    pub trait AutoImpl: ContractBase {}
```

Comment:
Leave the fully qualified version in place, this is how it gets generated by the macros